release-notes/opensearch-security-dashboards-plugin.release-notes-2.8.0.0.md

```diff
@@ -5,6 +5,7 @@ Compatible with Opensearch-Dashboards 2.8.0
 ### Bug Fixes
 
 * Fix the test cases to adapt the security password validation feature ([#1437](https://github.com/opensearch-project/security-dashboards-plugin/pull/1437))
+* Fixing dynamic tenancy changes for issues 1412 (#1419) ([#1459](https://github.com/opensearch-project/security-dashboards-plugin/pull/1459))
```

Comment:
```suggestion
* Fix dynamic tenancy changes for issues 1412 (#1419) ([#1459](https://github.com/opensearch-project/security-dashboards-plugin/pull/1459))
```

Reply:
It is labelled `Fixing` in the original PR: https://github.com/opensearch-project/security-dashboards-plugin/pull/1419